app/models/compute_attribute.rb

```diff
@@ -18,6 +18,10 @@ def method_missing(method, *args, &block)
     raise Foreman::Exception.new(N_('%s is an unknown attribute'), method)
   end
 
+  def respond_to?(method)
```

Comment:
not strictly related but we redefine `method_missing` few lines above so this should not be missing